lib/api/v3/work_packages/schema/work_package_schema_representer.rb

```diff
@@ -73,13 +73,27 @@ def create(work_package_schema, context)
           schema :description,
                  type: 'Formattable'
 
-          schema :start_date,
-                 type: 'Date',
-                 required: false
+          property :start_date,
+                   exec_context: :decorator,
+                   getter: -> (*) do
+                     representer = ::API::Decorators::PropertySchemaRepresenter
+                                   .new(type: 'Date',
+                                        name: WorkPackage.human_attribute_name(:start_date))
+                     representer.writable = represented.due_date_writable?
```

Comment:
@Azure7111 that should read 
`representer.writable = represented.start_date_writable?`

Although it shouldn't make any difference as of now. But I like that you distinguished between the two from the beginning.